frontend/src/slices/UserSlice.ts

```diff
@@ -88,4 +92,34 @@ export const createUserSlice: StateCreator<Store, [], [], UserSlice> = (
       toast.info(`Sent friend request: ${friendNetId}`);
     }
   },
+  async addWorksheet(season: Season, name: string) {
```

Comment:
FYI you don't need to add these API handles to the user context unless you need something about the user. The friends features are there because we do some client validations before we issue the request, but you don't have those (or at least not yet), so you can directly call the API functions.

Reply:
Amazing, thanks